src/main/java/roomescape/model/Reservation.java

```diff
@@ -1,28 +1,28 @@
 package roomescape.model;
 
-import java.time.LocalDateTime;
+import java.time.LocalDate;
 
 public class Reservation {
     private final Long id;
     private final String name;
-    private final LocalDateTime reservationTime;
+    private final LocalDate reservationDate;
+    private final ReservationTime reservationTime;
 
-    private Reservation(Long id, String name, LocalDateTime reservationTime) {
+    private Reservation(Long id, String name, LocalDate reservationDate, ReservationTime reservationTime) {
         this.id = id;
         this.name = name;
+        this.reservationDate = reservationDate;
         this.reservationTime = reservationTime;
     }
 
-    public static Reservation createReservation(Long id, String name, LocalDateTime reservationTime) {
-        return new Reservation(id, name, reservationTime);
+    public static Reservation createReservationWithoutId(String name, LocalDate reservationDate,
+                                                         ReservationTime reservationTime) {
+        return new Reservation(null, name, reservationDate, reservationTime);
     }
 
-    public static Reservation createReservationWithId(Long id, Reservation reservation) {
-        return new Reservation(id, reservation.getName(), reservation.reservationTime);
-    }
-
-    public static Reservation createReservationWithoutId(String name, LocalDateTime reservationTime) {
-        return new Reservation(null, name, reservationTime);
+    public static Reservation createReservationWithId(Long id, String name, LocalDate reservationDate,
```

Comment:
```suggestion
    public static Reservation of(Long id, String name, LocalDate reservationDate,
```
메서드명이 불필요하게 길어져도 가독성이 저하됩니다. 다른 메서드와 ReservationTime도 같이 살펴봐주세요~

Reply:
무조건 메서드 명으로 메서드에 대해 설명해야한다고 생각했습니다. 하지만 파랑의 추천 코드를 보고 메서드명 뿐만 아니라 파라미터, 리턴 타입으로 충분히 설명할 수 있다는 점을 알게되었습니다.
지금 다시보니 파라미터에서 제공하는 정보를 중복으로 메서드명으로 설명하여 가독성이 저하되는거 같습니다. 수정하도록 하겠습니다.